app/styles/app.scss

```diff
@@ -9,7 +9,7 @@ table {
   display: grid;
   grid-template:
     "sidebar  main" auto /
-    25rem 3fr 300px;
+    25rem 3fr;
```

Comment:
why did we need to make this change?

Reply:
The page looks weird 

before: 
<img width="453" alt="image" src="https://github.com/user-attachments/assets/40cb7385-c4a4-4198-84c5-a00e584938f7" />


after: 

<img width="453" alt="image" src="https://github.com/user-attachments/assets/9c3e9245-8c97-47f1-84ac-57ab88cb0eec" />